aea/agent_loop.py

```diff
@@ -43,20 +46,39 @@
 logger = logging.getLogger(__name__)
 
 
-class BaseAgentLoop(WithLogger, ABC):
+class AgentLoopException(AEAException):
+    """Exception for agent loop runtime errors."""
+
+
+class AgentLoopStates(Enum):
+    """Internal agent loop states."""
+
+    initial = None
```

Comment:
why `initial = None`? I haven't read the rest of the code, but considering the enumeration in isolation makes me think if there's an intrinsic reason why `initial` should be `None` and the other are strings. I feel it introduces asymmetry with other values.

Anyway, not really important, just wanted to share my thought.

Reply:
not sure this state is required, probably will remove it somewhen